components/src/core/ChannelValue/ChannelValue.tsx

```diff
@@ -46,7 +54,7 @@ const useStyles = makeStyles((theme: Theme) =>
             color: (props: ChannelValueProps): string => props.color,
         },
         icon: {
-            marginRight: theme.spacing(0.5),
+            marginRight: '0.35rem',
```

Comment:
```suggestion
            marginRight: '0.35em',
```

https://github.com/brightlayer-ui/angular-component-library/pull/331#discussion_r703687331

Reply:
@huayunh  In react for this example https://brightlayer-ui-components.github.io/react/?path=/story/components-channel-value--with-full-config font-size is 30px default. Because of this `margin-left: 0.35em` is 10.5px.
<img width="1174" alt="Screenshot 2022-01-17 at 8 58 03 PM" src="https://user-images.githubusercontent.com/10433274/149797942-444597f9-35e1-4b7e-8fca-a16f06692a5d.png">

Are we expecting this behaviour ?

Reply:
Yes, I am expecting the space to grow/shrink depending on the font size.